cmd/ak/cmd/triggers/create.go

```diff
@@ -6,20 +6,21 @@ import (
 	"github.com/spf13/cobra"
 
 	"go.autokitteh.dev/autokitteh/cmd/ak/common"
-	"go.autokitteh.dev/autokitteh/internal/backend/fixtures"
 	"go.autokitteh.dev/autokitteh/internal/kittehs"
 	"go.autokitteh.dev/autokitteh/internal/resolver"
 	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
 var (
 	call, event, filter, name, project, schedule string
-	data                                         map[string]string
+	webhook                                      bool
 )
 
 var createCmd = common.StandardCommand(&cobra.Command{
-	Use: `create -n name --call file:func [-p project] [-e env] -c connection [-E event] [-f filter] [--data key=value]...
-             create -n name --call file:func [-p project] [-e env] -s "schedule"`,
+	Use: `create -n name --call file:func [-p project] [-e env] -c connection [-E event] [-f filter]
+             create -n name --call file:func [-p project] [-e env] -s "schedule"
+  			 create -n name --call file:func [-p project] [-e env] --webhook
```

Comment:
There's nothing inherently wrong here, except that having 1 command that can create 3 different things depending on mutually-exclusive flags is not easy to comprehend unless you're already an expert in using our system.

In other words, our users will 100% struggle before they're able to use triggers effectively, or more likely give up before that.

At this points, it's better to create separate commands for these modes - it won't solve the underlying problem, but at least it would hide it better.

Reply:
we'll consider it in the future. for now i think it suffices.